manifests/g/GLPI-Project/GLPI-Agent/1.11/GLPI-Project.GLPI-Agent.locale.en-US.yaml

```
@@ -0,0 +1,81 @@
+Copyright: |-
+  Copyright 2006-2010 OCS Inventory contributors
+  Copyright 2010-2019 FusionInventory Team
+  Copyright 2011-2024 Teclib Editions
+# CopyrightUrl:
+ShortDescription: The GLPI Agent is a generic management agent. It can perform a certain number of tasks, according to its own execution plan, or on behalf of a GLPI server acting as a control point.
+Description: |-
+  The GLPI Agent is a generic management agent. It can perform a certain number of tasks, according to its own execution plan, or on behalf of a GLPI server acting as a control point.
+  This agent is based on a fork of FusionInventory agent and so works mainly like FusionInventory agent. It introduces new features and a new protocol to communicate directly with a GLPI server and its native inventory feature. Anyway it also keeps the compatibility with FusionInventory for GLPI plugin.
+Moniker: glpi-agent
+Tags:
+- glpi
+ReleaseNotes: |-
+  core:
+  * Prevent certificates overwriting during export from Windows Keystore
+  * Add new option to specify or disable Windows KeyStore support
+
+  inventory:
+  * fix #700: Add TacticalRMM Remote_Mgmt support for MacOSX
+  * fix #711: Update Bitdefender AV support on Windows, and also update enable support
+    on Windows Server OS.
+  * fix #716: Don't fail on regexp error while running from a user folder which contains
+    a parenthesis
+  * fix #748: Don't include lastloggeduser in hardware section in partial inventory
+    if users section is being deleted after no change detected
+  * Updated pci.ids to 2024.09.20 version
+  * Bump Inventory task version to 1.18
+
+  netdiscovery/netinventory:
+  * Skip Konica printers firmware with "Registered" set as version
+  * Enhanced Hikvision devices support
+  * Updated sysobject.ids
+  * Bump NetDiscovery task version to 6.4
+  * Bump NetInventory task version to 6.4
+
+  deploy:
+  * Avoid perl syntax error when running Powershell script from Deploy task on windows
+  * Bump Deploy task version to 3.3
+
+  proxy-server-plugin:
+  * Always evaluate only_local_store to yes when glpi_protocol is set and no server is
+    configured so default required format remains json
+  * Bump Proxy plugin version to 2.5
+
+  packaging:
+  * Update Windows MSI packing building process to use:
+    - OpenSSL 3.3.2
+    - libxml2 2.13.4
+  * Update MacOSX packages to use OpenSSL 3.3.2
+  * Windows MSI installer now supports AGENTMONITOR_NEWTICKET_SCREENSHOT as option to
+    configure if GLPI-AgentMonitor should always create a screenshot on new ticket.
+    The default set value is 1 and means to always create a screenshot.
+  * Updated Windows packages 7-Zip commandline tools to v24.08
+  * win32: Updated GLPI-AgentMonitor to v1.4.0
+ReleaseNotesUrl: https://github.com/glpi-project/glpi-agent/blob/HEAD/Changes
+PurchaseUrl: https://glpi-project.org/prices/
+# InstallationNotes:
+Documentations:
+- DocumentLabel: Documentation
+  DocumentUrl: https://glpi-project.org/documentation/

```

Documentation URL is wrong as it's the server part one, not agent related one. Better use the RTD documentation url.
```suggestion
  DocumentUrl: https://glpi-agent.readthedocs.io/
```